tests/golos.publication_rewards_tests.cpp

```diff
@@ -1083,8 +1092,10 @@ BOOST_FIXTURE_TEST_CASE(golos_delegators_test, reward_calcs_tester) try {
     const int divider = vest.make_asset(1).get_amount();
     const int min_remainder = delegation_min_remainder / divider;
     const auto amount = vest.make_asset(min_remainder);
+    auto vest_amount = vest.make_asset(150000000);
     uint16_t interest_rate = 3000;
-    uint8_t interest_rate0 = 0;
+    const uint16_t interest_rate_1prcnt = 100;
+    const uint16_t interest_rate_100prcnt = interest_rate_1prcnt * 100;
```

Comment:
you can use `golos::config::_100percent`. not sure, but there can also exist `golos::config::_1percent`